### PR TITLE
Schedules Test Case (2)

### PR DESCRIPTION
**Pull request by jorgeorpinel**
Improvements on previous PR per https://github.com/gonebusy/gonebusy-php-client/pull/23#issuecomment-294639298

### CODE REVIEW

File: test/Controllers/SchedulesTest.php [hunk 1]
```diff
@@ -170,19 +170,19 @@ private function bodyFromResponse($response, $type) {
                     $response->schedule->timeWindows[1]->recursBy,
                     $response->schedule->timeWindows[1]->startDate,
                     $response->schedule->timeWindows[1]->startTime,
-                    $response->schedule->timeWindows[1]->dateRecursBy,
+                    $response->schedule->timeWindows[1]->dateRecursBy?$response->schedule->timeWindows[1]->dateRecursBy:NULL,
```
Comment (alexagranov) [inline test/Controllers/SchedulesTest.php]: newline please

File: test/Controllers/SchedulesTest.php [hunk 2]
```diff
@@ -170,19 +170,19 @@ private function bodyFromResponse($response, $type) {
...
                     $response->schedule->timeWindows[1]->endDate,
                     NULL, // $response->schedule->timeWindows[1]->frequency,
                     NULL, // $response->schedule->timeWindows[1]->occurrence,
                     NULL // $response->schedule->timeWindows[1]->totalMinutes
                 );
             case 'updateSTW':
                 return new UpdateScheduleTimeWindowByIdBody(
-                    $response->schedule->timeWindows[1]->dateRecursBy,
+                    $response->schedule->timeWindows[1]->dateRecursBy?$response->schedule->timeWindows[1]->dateRecursBy:NULL,
```
Comment (alexagranov) [inline test/Controllers/SchedulesTest.php]: newline please

**Comment by alexagranov** [comment]
nice thanks!  just a couple of formatting comments.

File: test/Controllers/SchedulesTest.php (hunk 1 above)
Comment (jorgeorpinel) [inline test/Controllers/SchedulesTest.php]: Sure thing.

**Comment by jorgeorpinel** [comment]
Ready. I added indentation like there already was in line 151-3 though, lmk if I should roll all the cases back to single lines (with spaces between `?` and `:`). Thanks @alexagranov

**Pull request merged**